json_schema_fuzz/merging.py

```diff
@@ -1,9 +1,38 @@
 """Merging."""
+import copy
 
 
-def merge(schema_a, schema_b):
-    """Merge two JSON schemas."""
-    return {
-        **schema_a,
-        **schema_b,
-    }
+def merge(
+    a: dict[any, any],
+    b: dict[any, any],
+    path: list = None,
+    update: bool = False
```

Comment:
I think the `update` business is not necessary. For a general dict merging utility, fine, but when merging JSON schemas there is one correct behavior, so this should not be configurable.